Mathlib/Order/Hom/WithTopBot.lean

```diff
@@ -344,7 +344,6 @@ variable [Lattice α] [Lattice β] [Lattice γ]
 protected def withTop (f : LatticeHom α β) : LatticeHom (WithTop α) (WithTop β) :=
   { f.toInfHom.withTop with toSupHom := f.toSupHom.withTop }
 
--- Porting note: `simps` doesn't generate those
```

Comment:
This does feel like a regression. Would adding `@[simps apply coe]` to `withTop` generate this lemma?

Reply:
I tried, and it didn't... should I revert these?

Reply:
It is a relatively minor issue. If there are other parts of the library where these kinds of lemmas are also done by hand without a comment, then I'm okay with removing the porting note here.

Reply:
Hmm, I'll play on the safe side and revert.